src/index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Challenge</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    
+
+    <!-- Container Starts-->
+    <div class="container">
+
+        <!-- Header -->
+        <header class="header">
+            <div class="header-logo">
+                <img class="header-logo-img"src="../assets/twitter-color-svgrepo-com.svg" alt="logo">
+            </div>
+            <div class="home-bar"><p><strong>Home</strong></p></div>
+            <div class="header-icon">
+                <img class="header-icon-img" src="../assets/sparkles-svgrepo-com.svg" alt="logo">
+            </div>
+            <div class="search-twitter-bar">
+                <div class="magnifying-logo">
+                    <img class="magnifying-logo-img"src="../assets/search-alt-svgrepo-com.svg" alt="search">
+                </div>
+                <div class="search-text"><p>Search Twitter</p></div>
+            </div>
```

Comment:
You are using alt in img tags it is really good I like it good job :)

Reply:
Thanks Nazar :)